### PR TITLE
fix(scheduler): update runtime label to support global service in the namespace

### PR DESCRIPTION
**Pull request by Muzry**
#### What type of PR is this?
/kind bug
#### What this PR does / why we need it:
- fix(scheduler): update runtime label to support global service in the namespace

#### Which issue(s) this PR fixes:

Fixes #

#### Specified Reivewers:
/assgin @luobily @sixther-dc 

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation usage docs, etc.:

```docs

```

### CODE REVIEW

File: modules/scheduler/executor/plugins/k8s/namespace/namespace.go [hunk 1]
```diff
@@ -71,7 +71,7 @@ func (n *Namespace) Create(ns string, labels map[string]string) error {
 	if !resp.IsOK() {
 		return errors.Errorf("failed to create namespace, ns: %s, statuscode: %v, body: %v", ns, resp.StatusCode(), b.String())
 	}
-	logrus.Infof("succeed to create namespace, ns: %s", ns)
+	logrus.Infof("succeed to create namespace, ns %s from module %s", ns, module)
```
Comment (luobily) [inline modules/scheduler/executor/plugins/k8s/namespace/namespace.go:74]: parameters should not only be used to print logs.
pls delete this param (module).

**Review by luobily — CHANGES_REQUESTED** [review]

File: modules/scheduler/executor/plugins/k8s/namespace/namespace.go (hunk 1 above)
Comment (Muzry) [reply modules/scheduler/executor/plugins/k8s/namespace/namespace.go]: already  modify to 
```suggestion
logrus.Infof("succeed to create namespace, ns: %s", ns)
```

**Review by luobily — APPROVED** [review]
/lgtm

**Pull request merged**